packages/agoric-cli/test/snapshots/test-inter-cli.js.md

```diff
@@ -51,31 +51,37 @@ Generated by [AVA](https://avajs.dev).
 
     `Usage: agops inter bid by-price [options]␊
     ␊
-    Print an offer to bid collateral by price.␊
+    Place a bid on collateral by price.␊
     ␊
     Options:␊
-      --price [number]            bid price␊
-      --giveCurrency [number]     Currency to give␊
-      --wantCollateral [number]   Collateral expected for the currency␊
-      --collateralBrand [string]  Collateral brand key (default: "IbcATOM")␊
-      --offerId [number]          Offer id (default: "bid-978307200000")␊
-      -h, --help                  display help for command`
+      --from <address>        wallet address literal or name␊
+      --give <amount>         IST to bid␊
+      --desiredBuy <amount>   max Collateral wanted (default: "1_000_000IbcATOM")␊
+      --wantMinimum <amount>  only transact a bid that supplies this much␊
```

Comment:
Is there a way to show that `wantMinimum` is not required?

Reply:
yes, but the part of my brain that writes good short descriptions is exhausted.

I hope we can address it later, or in end-user docs #6889 .